packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -250,6 +258,13 @@ class TextStyle extends Diagnosticable {
   final bool inherit;
 
   /// The color to use when painting the text.
+  /// 
+  ///  If [foreground] is specified, this value must be null.  [color] is shorthand for
```

Comment:
To avoid starting the sentence with a lowercase letter, consider `The [color] property is...`